### ISSUE TITLE
diagnostics fails to get some results against cloud

### ISSUE THREAD

**Issue by kiju98** [elastic/support-diagnostics#275]
Diagnostics version: 7.0.5

Someone wants to automate running diagnostics.sh for some reason I don't know yet.
They tried to run diagnostics.sh but only got partial results with some errors.
I suspect the timeout in diagnostics.sh is too short for a cluster in the cloud and diagnostics.sh gets the results from some REST commands and doesn't get from some other commands.

I tried the following command again my test cluster in the cloud and could reproduce the symptom:

```
./diagnostics.sh --host b087af4041544ca4a7b0e842e1eed913.us-west-2.aws.found.io --port 9243 --type remote -u elastic --ptp <password> --ssl
```

Will you look into it, please?

**Comment by kiju98** [comment]
I can ask them but I used the `elastic` user which is the built-in superuser for the cluster and could reproduce it.

**Comment by kiju98** [comment]
They used 
```
./diagnostics.sh --host
<deployment ID>.us-west-2.aws.found.io --port 9243 -u elastic --ptp <password> --type remote --ssl
```

So they used the `elastic` account to run the diag.

**Comment by nemonster** [comment]
Fixed in version 7.0.6.

**Comment by nemonster** [comment]
Yes, I went ahead and built deployed before I actually merged the code. It's up to date now.
